tests/test-widget-show-instance-in-rest.php

```
@@ -0,0 +1,71 @@
+/**
+ * WidgetShowInstanceInRest test case.
+ */
+class WidgetShowInstanceInRestTest extends WP_UnitTestCase {
+
+	/**
+	 * 検証対象となる ExUnit ウィジェットクラス名の一覧。
+	 *
+	 * @return string[] ウィジェットクラス名の配列。
+	 */
+	public function widget_class_provider() {
+		return array(
```

PHPStan (2.2.2)

[error] Method WidgetShowInstanceInRestTest::widget_class_provider() should return array<string> but returns array<int, list<string>>.


(return.type)